AmplifyPlugins/DataStore/AWSDataStoreCategoryPlugin/Sync/SubscriptionSync/AWSIncomingEventReconciliationQueue.swift

```diff
@@ -89,9 +93,11 @@ final class AWSIncomingEventReconciliationQueue: IncomingEventReconciliationQueu
         case .mutationEvent(let event):
             eventReconciliationQueueTopic.send(.mutationEvent(event))
         case .connected(let modelName):
-            reconciliationQueueConnectionStatus[modelName] = true
-            if reconciliationQueueConnectionStatus.count == reconciliationQueues.count {
-                eventReconciliationQueueTopic.send(.initialized)
+            connectionStatusOperationQueue.async {
```

Comment:
Should protect the assignment in `onReceiveCompletion`, as well as `cancel` and any other assignments of instance variables that I missed in my scan.